src/app/core/components/public-settings/public-settings.component.ts

```diff
@@ -12,11 +15,32 @@ export class PublicSettingsComponent implements OnInit {
   @Input() public archive: ArchiveVO;
   public updating: boolean = false;
   public allowDownloadsToggle: number = 0;
+  public supportLink:string = 'https://permanent.zohodesk.com/portal/en/newticket'
```

Comment:
Can you run prettier on this file? This line sticks out to me a bit, and I think prettier should have fixed that.

Reply:
done